imjoy/imjoyPluginEngine.py

```diff
@@ -62,10 +63,16 @@ def get_drives():
 except ImportError:
     from queue import Queue, Empty  # python 3.x
 
+# read version information from file
+HERE = pathlib.Path(__file__).parent
+version_info = json.loads((HERE / ".." / "VERSION").read_text())
+
+__version__ = version_info["version"]
+__api_version__ = version_info["api_version"]
```

Comment:
If `__api_version__` is only intended to be used within this file, it should just be a constant, I think.

Reply:
I was trying to make it looks like __version__, so I should rename it as `API_VERSION`?

Reply:
Yes, I think so, unless we have a specific use case for the magic module attribute that requires it to be that.